src/static/css/link.css

```diff
@@ -0,0 +1,37 @@
+a {
+    color: var(--fl-linkColor, var(--link-color));
+    text-decoration: underline;
+}
+
+a:hover {
+    color: var(--fl-linkColor, var(--link-hover-color));
+}
+
+a:active {
+    background: var(--fl-bgColor, var(--link-active-bg));
+    box-shadow: 0 0 0 0.3em var(--fl-bgColor, var(--link-active-bg));
```

Comment:
I think this should probably be expressed in a unit other than `em` — probably `rem`. Unless you want a much bigger padding around an h1 link than a paragraph link which I don't think you do.